pkg/golinters/usetesting/testdata/usetesting.go

```diff
@@ -11,7 +11,7 @@ func Test_osMkdirTemp(t *testing.T) {
 }
 
 func Test_osSetenv(t *testing.T) {
-	os.Setenv("", "")
+	os.Setenv("", "") // want `s\.Setenv\(\) could be replaced by t\.Setenv\(\) in .+`
```

Comment:
Any reason to have this instead of 


```suggestion
	os.Setenv("", "") // want `os\.Setenv\(\) could be replaced by t\.Setenv\(\) in .+`
```